scripts/build_gh_pages

```diff
@@ -12,7 +12,7 @@ readonly SCRIPTS_DIR="$(dirname "$0")"
 # shellcheck source=scripts/common
 source "$SCRIPTS_DIR/common"
 
-readonly TARGET_DIR="${1:-}"
+readonly TARGET_DIR="$(cd "${1:-}" && pwd)"
```

Comment:
Could you put a comment explaining what this produces for a missing $1 (on first glance I assumed it would give `$HOME` but it actually gives `$PWD`&hellip;).